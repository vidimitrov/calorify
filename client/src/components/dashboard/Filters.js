import styled from 'styled-components';

const Filters = styled.div`
  display: flex;
  align-items: center;
  flex-direction: ${props => (props.visible ? 'row' : 'row-reverse')};
  padding: ${props => (props.visible ? '16px 32px 0' : '8px 32px 0')};
  margin-bottom: 20px;
  text-align: right;
`;

export default Filters;
