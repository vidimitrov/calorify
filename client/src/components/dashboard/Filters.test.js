import React from 'react';
import renderer from 'react-test-renderer';
import 'jest-styled-components';
import Filters from './Filters';

it('Dashboard/Filters component renders correctly', () => {
  const tree = renderer.create(<Filters />).toJSON();
  expect(tree).toMatchSnapshot();
});
