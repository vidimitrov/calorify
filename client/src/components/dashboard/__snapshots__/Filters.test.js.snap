// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Dashboard/Filters component renders correctly 1`] = `
.c0 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  -webkit-flex-direction: row-reverse;
  -ms-flex-direction: row-reverse;
  flex-direction: row-reverse;
  padding: 8px 32px 0;
  margin-bottom: 20px;
  text-align: right;
}

<div
  className="c0"
/>
`;
